.gitignore

```diff
@@ -7,3 +7,6 @@
 /vendor/
 .phpunit.result.cache
 .phpcs-cache
+.buildpath
+.project
+.settings/org.eclipse.php.core.prefs
```

Comment:
Please add this to your local Git configuration and undo these changes here.

Reply:
Yea I was using Zend studio then which made it a pain. I'll fix it this evening.

Reply:
@froschdesign 
Should be fixed now.